build/release/cdn.js

```diff
@@ -68,12 +69,15 @@ function makeArchives( Release, callback ) {
 
 		console.log( "Creating production archive for " + cdn );
 
-		var sum,
+		var i,
+			sum,
+			result,
```

Comment:
The style usually used in other places is to put all uninitialized variable declarations in one line:
```suggestion
		var i, sum, result,
```

Reply:
build/release/cdn.js